tests/helpers.py

```diff
@@ -0,0 +1,20 @@
+import asyncio
+from unittest import mock
+
+
+def make_coro_mock():
```

Comment:
@mauricioabreu I'm no sure about how to use fixtures on this case. Actually, the values of fixtures tha are inject, no the fixture itself. So, no sure if fixtures can help us here. 

I think the right thing to do would be to inject the object already mocked, but that will need one fixture per mock, not sure if it's a good idea also.